src/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/gather_weights_decompression.hpp

```diff
@@ -57,5 +57,33 @@ class GatherWeightsDecompression : public testing::WithParamInterface<GatherWeig
     void SetUp() override;
 };
 
+/*
+ *    Weights(FP16/BF16)
+ *          |
+ *    Convert(F32) Indices(I32)
+ *          \      /
+ *           Gather
+ */
+using GatherWeightsDecompressionWithoutScaleParams = std::tuple<std::string,  // Device name
+                                                                GatherDecompressionShapeParams,
+                                                                ov::element::Type,   // data type
+                                                                ov::element::Type>;  // output type
+
+class GatherWeightsDecompressionWithoutScale
```

Comment:
Could you please pick out a common part from both classes? For example, `GatherWeightsDecompression::check_results` and `GatherWeightsDecompression::generate_inputs` can be used in both tests. We could create a base class for that

Reply:
Sure, applied, Thanks Vladislav!